### PR TITLE
[WIP] Command scopes fixes: #2061, #2072, #2075

### LINKED ISSUE #2061: No Unsigned Transaction screen is shown if send tx in chat with 2 members [feature/command-scopes]
### Description

*Type*: Bug

*Summary*: 
User can't send a transaction in group chat with 2 members because Unsigned transaction screen is not shown after tap on send button. Instead, same chat is shown with nothing in the input field

#### Expected behavior
If send command gets valid parameters and user taps on send button then Unsigned transaction screen is shown
#### Actual behavior
same chat is shown with nothing in the input field

### Reproduction
(from 00:41) https://app.testfairy.com/projects/4803622-status/builds/6783719/sessions/8/?accessToken=ONT3syhpY2Zo72TKoD0cPycUZQM
- Open Status
- Create group chat with 1 member, so in total there are 2 members: you and e.g. Jarrad 
- In group chat tap on send command. Note that there is no suggestions shown
- type valid amount and tap on send button

### Additional Information
* Status version: feature/command-scopes  0.9.10-175-g9a4ef51a
* Operating System: 
Real device iPhone 7 plus, iOs 11.0.2
Real device Google Pixel XL, Android 8.0

### PR DESCRIPTION
**Pull request by alwx**
Fixes #2061, #2072, #2075

### CODE REVIEW

**Review by rasom — APPROVED** [review]

File: .re-natal [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "StatusIm",
   "interface": "reagent",
-  "androidHost": "localhost",
+  "androidHost": "10.0.3.2",
```
Comment (yenda) [inline .re-natal:4]: don't include these files

**Review by yenda — CHANGES_REQUESTED** [review]

**Comment by yenda** [comment]
Is this going to be squashed in the other PR ? Why is it in a separate PR ?

File: .re-natal (hunk 1 above)
Comment (alwx) [reply .re-natal:4]: it's still in progress, it doesn't suppose to be merged right now

Comment (yenda) [reply .re-natal:4]: my bad, makes me realize the status we usually put in the PR description (ready/wip) is redundant with the column in zenhub

**Pull request merged**